tajo-catalog/tajo-catalog-client/src/main/java/org/apache/tajo/catalog/AbstractCatalogClient.java

```diff
@@ -754,26 +801,19 @@ public final FunctionDesc getFunction(final String signature, FunctionType funcT
       builder.addParameterTypes(type);
     }
 
-    FunctionResponse response = null;
     try {
       final BlockingInterface stub = getStub();
-      response = stub.getFunctionMeta(null, builder.build());
-    } catch (ServiceException se) {
-      throw new RuntimeException(se);
-    }
-
-    if (isThisError(response.getState(), ResultCode.UNDEFINED_FUNCTION)) {
-      throw new UndefinedFunctionException(signature, paramTypes);
-    } else if (isThisError(response.getState(), ResultCode.AMBIGUOUS_FUNCTION)) {
-      throw new AmbiguousFunctionException(signature, paramTypes);
-    }
+      final FunctionResponse response = stub.getFunctionMeta(null, builder.build());
```

Comment:
If there are several functions of the given name, getFunctionMeta() always throws a function which has the nearest argument among them instead of throwing ```AmbiguousFunctionException```.